src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -374,9 +374,9 @@ export class StudySummaryTab extends React.Component<
                     props.onResetSelection = this.handlers.onValueSelection;
                 }
                 props.onChangeChartType = this.handlers.onChangeChartType;
-                props.getData = () =>
-                    this.store.getChartDownloadableData(chartMeta);
-                props.downloadTypes = ['Data'];
+                props.getData = chartType =>
+                    this.store.getChartDownloadableData(chartMeta, chartType);
```

Comment:
Minor, but should we name the param `dataType`?

```suggestion
                props.getData = dataType =>
                    this.store.getChartDownloadableData(chartMeta, dataType);
```